website/docs/installation/docker-compose.md

```diff
@@ -4,6 +4,10 @@ title: Docker Compose installation
 
 This installation method is for test-setups and small-scale production setups.
 
+:::info
+You can also view a [video walk-through](https://youtu.be/owk1a_1xYe4) of the installation process on Docker (with bonus details about email configuration and other important options).
```

Comment:
Weak suggestion: `[view a video walkthrough]`.  The question of whether or not a link is a suggestion to "do a thing" or "get a thing" is always questionable, but I have a preference for the link-as-verb because a video commits me to a period of time, so the link is connected to the thought _I'm going to spend some time viewing a video_.